methods.py

```diff
@@ -645,7 +645,7 @@ def detect_darwin_sdk_path(platform, env):
 
     if not env[var_name]:
         try:
-            sdk_path = subprocess.check_output(['xcrun', '--sdk', sdk_name, '--show-sdk-path']).strip()
+            sdk_path = subprocess.check_output(['xcrun', '--sdk', sdk_name, '--show-sdk-path']).strip().decode()
```

Comment:
That might break compilation with Python 2. Check `compat.py` for methods that can be used to be compatible with both Python 2 and Python 3 (seems like `decode_utf8` might be what you need here).

Reply:
Thank you for checking. I fixed it and rebased.